storage/client/queries.go

```diff
@@ -46,14 +46,17 @@ func (qf QueryFactory) TransactionsQuery() string {
 	return fmt.Sprintf(`
 		SELECT 
 				%[1]s.transactions.block as block,
+				%[1]s.transactions.tx_index as tx_index,
 				%[1]s.transactions.tx_hash as tx_hash,
 				%[1]s.transactions.sender as sender,
 				%[1]s.transactions.nonce as nonce,
 				%[1]s.transactions.fee_amount as fee_amount,
 				%[1]s.transactions.method as method,
 				%[1]s.transactions.body as body,
-				%[1]s.transactions.code as code
+				%[1]s.transactions.code as code,
+				%[1]s.blocks.time as time
```

Comment:
I don't think the MVP mocks have it anywhere, so let's add it when/if needed.

I am actually itching to remove some of the filters we already have. Every filter needs a new index (and they make inserts/updates expensive). Some of the filters we have _don't_ have a filter, so they will result in horribly slow queries that will gunk up our DB. Again, the more filters we have, the more likely a user can create an expensive query by using that filter (or a creative filter combo), either because they're actively trying to be evil, or (more likely) because we didn't prepare sufficiently for that kind of query.

To your point, as a user, I do want a time filter in the GUI. So we might well end up adding it eventually.

Reply:
Agreed on both points; will add to the product meeting agenda.